html/main.js

```diff
@@ -0,0 +1,179 @@
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"description\">\n" +
+        "                                    <div class=\"ui transparent fluid input\">\n" +
+        "                                        <input type=\"text\" placeholder=\"Your command ...\" value=\"" + command + "\">\n" +
+        "                                        <div class=\"ui left pointing label olive\" style=\"display: none\">Keep it short\n" +
+        "                                        </div>\n" +
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"description\">\n" +
+        "                                    <div class=\"ui transparent fluid input\">\n" +
+        "                                        <input type=\"text\" placeholder=\"Output text ...\" value=\"" + output + "\">\n" +
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"ui two buttons\">\n" +
+        "                                    <div class=\"ui basic grey button disable\">Disable</div>\n" +
+        "                                    <div class=\"ui basic red button delete\">Delete</div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                        </div>";
+
+    let existingDiv = $("#cards-parent > div").last();
+    if (existingDiv.length) {
+        existingDiv.after(newcard);
+    } else {
+        cardsParent.append(newcard);
+    }
+
+    if (modalUi.modal("is active")) {
+        modalUi.modal("hide");
+    }
+}
+
+modalUi.modal("setting", "transition", "horizontal flip").modal({
+    closable: true,
+    onApprove() {
+        return false;
+    },
+    onShow() {
+        modalForm.form("clear");
+    }
+});
+
+function validateModal(form) {
+    // If title empty, return false
+    let inputTitle = form.find("input[name ='title']").val();
+    if (inputTitle === "") {
+        form.focus();
+        return false;
+    }
+
+    // Else call methods below
+    inputDescription = form.find("input[name ='description']").val();
+    inputCommand = form.find("input[name ='command']").val();
+    inputOutput = form.find("input[name ='output']").val();
+
+    sendNewCommand(inputTitle, inputDescription, inputCommand, inputOutput);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['sendNewCommand' was used before it was defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)